dist/src/main/java/io/camunda/application/StandaloneIdentityMigration.java

```diff
@@ -28,7 +28,7 @@ public static void main(final String[] args) {
     final SpringApplication application =
         new SpringApplicationBuilder()
             .logStartupInfo(true)
-            .web(WebApplicationType.SERVLET)
+            .web(WebApplicationType.NONE)
```

Comment:
not needed anymore as we don't rely on conditions on the default service configuration